src/index.jsx

```diff
@@ -40,4 +41,12 @@ initialize({
     appMessages,
     paragonMessages,
   ],
+  handlers: {
+    config: () => {
+      mergeConfig({
+        LEARNING_BASE_URL: process.env.LEARNING_BASE_URL,
+        DISPLAY_FEEDBACK_BANNER: process.env.DISPLAY_FEEDBACK_BANNER || 'false',
+      }, 'DiscussionsConfig');
+    },
+  },
 });
```

Comment:
Why do we need this? I am already using env variables from edx-internal and I don't find the use of **`mergeconfig`** so I just skipped it before.

Reply:
https://github.com/openedx/frontend-platform#application-configuration

 i got its usage.